pootle/core/search/backends/elasticsearch.py

```diff
@@ -26,6 +28,26 @@
 logger = logging.getLogger(__name__)
 
 
+def filter_hits_by_distance(hits, source_text, max_distance=0.3):
+    """Returns ES `hits` filtered according to their Levenshtein distance
+    to the `source_text`.
+
+    Any hits with a normalized distance value (0..1) higher than
+    `max_distance` will be discarded. It's assumed that `hits` is already
+    sorted from higher to lower score.
```

Comment:
Leaving that post-processing change it seems that we can rely on [elasticsearch returning the results sorted by relevance](https://www.elastic.co/guide/en/elasticsearch/guide/current/sorting.html).